CMakeLists.txt

```diff
@@ -114,7 +114,9 @@ option (WITH_DOXYGEN                        "Generate API Reference with Doxygen
 option (WITH_HEADERS                        "Install header files (to build samples)"               ON)
 option (WITH_SAMPLES                        "Install sample files (requires headers)"               ON)
 option (WITH_TESTS                          "Install unit tests and benchmarks"                     ON)
-option (WITH_PYTHON                         "Build Python bindings"                                 ON)
+option (WITH_PYTHON2_BINDINGS               "Build Python bindings for Python 2"                    ON)
```

Comment:
I was thinking of just changing the help text, not the option. This is cleaner but we must update build instructions now.

Reply:
I checked. We don't use WITH_PYTHON in the instruction. We use the default ON value.

Reply:
Ok cool, thanks.